### ISSUE TITLE
--sort-by CLI argument doesn't work

### ISSUE THREAD

**Issue by rrrix** [mhipszki/eslint-formatter-summary#28]
I ran it copypasta as in the README:

```
src $ eslint -f summary . -- --sort-by rule


Oops! Something went wrong! :(

ESLint: 6.8.0.

No files matching the pattern "--sort-by" were found.
Please check for typing mistakes in the pattern.
```

After reading the [ESLint docs on Custom Formatters](https://eslint.org/docs/developer-guide/working-with-custom-formatters) it doesn't seem like this is even supported by ESLint. Am I missing something?

Is there another way to sort output? Currently it appears random / nondeterministic. 

It's not a _major_ problem since piping to `| sort -k 2` (for example) does what I want, albeit without the fancy colors. 

Thanks for the awesome package :)

**Comment by mhipszki** [comment]
thanks @rrrix I'm looking into this issue to see if there's a fix for that

**Comment by mhipszki** [comment]
Right, so the CLI options `--sort-by` etc. suggested in the README apparently don't work anymore, we'll need to find another way to provide this functionality

Currently the recommended way in the [ESLint docs on Custom Formatters](https://eslint.org/docs/developer-guide/working-with-custom-formatters#passing-arguments-to-formatters) is to use env vars, which is quite awkward imo 😕 

I think the _non-determinism_ of the listed failing rules in the output is due to how ESLint is giving us the list of files and their errors.

I think both the colours and the sorting functionality is helpful especially in bigger projects, so I'll have a look if there are other options than the env vars otherwise I'll update the formatter to use them instead of the CLI options.

**Comment by Camo30** [comment]
Same issue here. May you could consider about applying a default sorting, for example by rule

**Comment by mhipszki** [comment]
> Same issue here. Maybe you could consider about applying a default sorting, for example by rule

thanks for the suggestion, I'll have a look and come back to you @Camo30

**Comment by mhipszki** [comment]
@rrrix fyi the sorting has been fixed now using env vars in #31 

@Camo30 fyi default sorting has been added in #32 

both features have been released in v1.1.0 🎉
